lib/lambda_ethereum_consensus/beacon/beacon_chain.ex

```diff
@@ -60,7 +60,10 @@ defmodule LambdaEthereumConsensus.ForkChoice do
 
     children = [
       {LambdaEthereumConsensus.ForkChoice.Store, {anchor_state, anchor_block}},
-      {LambdaEthereumConsensus.ForkChoice.Tree, []}
+      {LambdaEthereumConsensus.Beacon.PendingBlocks, []},
+      {LambdaEthereumConsensus.Beacon.SyncBlocks, []},
+      {LambdaEthereumConsensus.P2P.IncomingRequests, []},
+      {LambdaEthereumConsensus.P2P.GossipSub, []},
```

Comment:
in a follow up, we can calculate the fork digest from the anchor state and pass it down to the gossipsub and incoming requests to subscribe to the topics with the proper digest.

Eventually we will have to create a more complex logic, because when the fork changes we have to subscribe to the new fork topics.